configs/knot_raspberrypi0w_defconfig

```diff
@@ -57,6 +57,7 @@ BR2_PACKAGE_KNOT_CONTROL=y
 BR2_PACKAGE_KNOT_WEB=y
 BR2_PACKAGE_KNOT_FOG=n
 BR2_PACKAGE_KNOT_FOG_CONNECTOR=y
+BR2_PACKAGE_KNOT_BABELTOWER=y
```

Comment:
I would not add this package on this file.

Reply:
Done.